There's no archives in /bin or /usr/bin, so don't let those get
picked up by file name matching rules.

Upstream-status: Still local/experimental.

Signed-off-by: Peter Seebach <peter.seebach@windriver.com>

diff --git a/pkg-diff.sh b/pkg-diff.sh
index e77f4ab..915d7a7 100755
--- a/pkg-diff.sh
+++ b/pkg-diff.sh
@@ -288,6 +288,10 @@ check_single_file()
 {
   local file="$1"
   case $file in
+    /bin/*|/usr/bin/*)
+       # Handled by file type check. If this isn't here, the "cpio.cpio"
+       # alternative from cpio gets treated as a malformed cpio archive.
+       ;;
     *.spec)
        sed -i -e "s,Release:.*$release1,Release: @RELEASE@," old/$file
        sed -i -e "s,Release:.*$release2,Release: @RELEASE@," new/$file
